Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL org.prx.app="yes"
 LABEL org.prx.spire.publish.ecr="RAILS_APP"
 
 RUN apk -U upgrade && apk add --no-cache \
-    tzdata postgresql-dev postgresql-client build-base bash coreutils git
+    tzdata postgresql-dev postgresql-client build-base bash coreutils git nodejs
```

Comment:
For some reason, this is required by execjs, which is required by autoprefixer-rails, which is part of bootstrap.

I could have picked a [different runtime](https://github.com/rails/execjs) instead ... but I sort of figured we'd need node in here at some point (for linting).